### PR TITLE
Allow command console with superheavy/small cockpit

### PR DESCRIPTION
**Pull request by binaryspica**
For implement Superheavy/Small Command Console. It is related with [a pull request of MegaMek.](https://github.com/MegaMek/megamek/pull/1477)

### CODE REVIEW

**Comment by binaryspica** [comment]
Also I have a question. Is normal Command Console need to be allowed to the industrialmechs with Advanced Fire Control?

File: src/megameklab/com/ui/Mek/tabs/StructureTab.java [hunk 1]
```diff
@@ -293,6 +293,32 @@ private void resetSystemCrits() {
...
+                getMech().addSuperheavyIndustrialCockpit();
+                getMech().setArmorType(
+                        EquipmentType.T_ARMOR_INDUSTRIAL);
+                break;
+            case Mech.COCKPIT_SUPERHEAVY_COMMAND_CONSOLE:
+                clearCritsForCockpit(false, true);
+                getMech().addSuperheavyCommandConsole();
+                break;
+            case Mech.COCKPIT_SMALL_COMMAND_CONSOLE:
+                clearCritsForCockpit(True, true);
```
Comment (sixlettervariables) [inline src/megameklab/com/ui/Mek/tabs/StructureTab.java]:
```suggestion
                clearCritsForCockpit(true, true);
```

Comment (binaryspica) [reply src/megameklab/com/ui/Mek/tabs/StructureTab.java]: Thanks. I am bad at typo....

**Comment by binaryspica** [comment]
> As noted on the companion PR for MegaMek, the industrial cockpit type is for IMs without advanced fire control. Since advanced fire control is required for a cockpit command console, industrial cockpit with command console is not a legal combination.

Thanks for note this. I fixed the issue.

**Pull request merged**